cpp/oneapi/dal/backend/interop/table_conversion.hpp

```diff
@@ -253,6 +253,9 @@ inline table convert_from_daal_table(const daal::data_management::NumericTablePt
 #ifdef ONEDAL_DATA_PARALLEL
 inline daal::data_management::NumericTablePtr convert_to_daal_table(const sycl::queue& queue,
                                                                     const table& table) {
+    if (!table.has_data()) {
+        return daal::data_management::NumericTablePtr{};
+    }
```

Comment:
Conversion of empty oneDAL table typically indicates error on the caller side. What kind of situation you are trying to handle by introducing this branch?

Reply:
In this case this check fails but in fact DAAL backend requires empty table of labels for the search problem